web/src/routes/OrganizationsList/Organization/Tabs/Settings/Settings.tsx

```diff
@@ -266,6 +26,15 @@ export default function Settings(props: SettingsProps) {
       visible: true,
     },
     {
+      name: 'Billing Information',
+      id: 'billinginformation',
+      content: <BillingInformation organizationName={props.organizationName} />,
+      visible: quayConfig?.features?.BILLING,
+    },
+  ];
+
+  if (quayConfig?.features?.AUTO_PRUNE) {
```

Comment:
The `if` condition here is not required as visible prop [here](https://github.com/quay/quay/pull/2151/files#diff-e9a6c92a9f0242880ababdad54680c0dddca82d57a05ef44a564e8629108d388R46) does the same.

Reply:
Resolved